src/components/Activity.vue

```diff
@@ -164,6 +175,20 @@ function getExpectedTime(): void {
   emit('update:expectedTime', round(expectedTime));
 }
 
+function getFactorizedExpectedTime(): void {
+  if (!get(settings).useFactor) {
+    return;
+  }
+
+  if (props.mostLikely === 0 && props.pessimistic === 0) {
```

Comment:
doesn't that cause the time not to update, if ML or P is 0 ? Why is that check here? seems to make more sense in calcExpectedTime ? if at all?

Reply:
The reason behind this check is to only update if most likely and pessimistic are filled. A calculation does not really make sense without those two numbers. Whereas optimistic might be 0 in some cases. This was just copied from the calcExpectedTime.